foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/CountAction.java

```diff
@@ -77,7 +77,7 @@ public ActionResponse execute(CountRequest parameter) throws QueryStoreException
         try {
             if (parameter.isDistinct()){
                 SearchRequestBuilder query = getConnection().getClient()
-                        .prepareSearch(ElasticsearchUtils.getIndices(parameter.getTable()))
+                        .prepareSearch(ElasticsearchUtils.getIndices(parameter.getTable(), parameter))
```

Comment:
since parameter is already passed, no need to pass parameter.getTable() as a different method parameter.